packages/blockchain-extension/extension/debug/FabricDebugConfigurationProvider.ts

```diff
@@ -86,30 +86,39 @@ export abstract class FabricDebugConfigurationProvider implements vscode.DebugCo
                 return;
             }
 
-            const runtimes: LocalEnvironment[] = LocalEnvironmentManager.instance().getAllRuntimes();
-
             // At the moment, we only want the user to debug with 1-org environments.
             // See https://github.com/IBM-Blockchain/blockchain-vscode-extension/issues/1995 for the issue of supporting 2-org environments.
             const oneOrgRuntimes: string[] = [];
-            for (const _runtime of runtimes) {
-                if (_runtime.numberOfOrgs === 1) {
-                    const runtimeName: string = _runtime.getName();
-                    oneOrgRuntimes.push(runtimeName);
+            const environmentEntries: FabricEnvironmentRegistryEntry[] = await FabricEnvironmentRegistry.instance().getAll(true); // Get only local entries
+
+            const oneOrgEntries: FabricEnvironmentRegistryEntry[] = environmentEntries.filter((entry: FabricEnvironmentRegistryEntry) => {
+                return entry.numberOfOrgs === 1 && entry.environmentType === EnvironmentType.LOCAL_ENVIRONMENT;
```

Comment:
you don't need the check on the type as you already only got the local ones when you did getAll